core/controllers/feedback.py

```diff
@@ -58,7 +59,12 @@ class ThreadHandler(base.BaseHandler):
 
     def get(self, exploration_id, thread_id):
         self.values.update({
-            'messages': feedback_services.get_messages(thread_id)})
+            'messages': feedback_services.get_messages(".".join((
+                exploration_id, thread_id)))})
+        suggestion = feedback_services.get_suggestion(exploration_id, thread_id)
+        if suggestion is not None:
```

Comment:
I think maybe just return a dict with two keys in both cases. The value for 'suggestion' can be None -- which translates to null in JS.